proto/proto/wallet.proto

```diff
@@ -4,58 +4,29 @@ package penumbra.wallet;
 import "transaction.proto";
 import "crypto.proto";
 
+// The wallet protocol is used by a wallet client, who wants to do some
+// transaction-related actions, to request data from a wallet service, which is
+// responsible for synchronizing and scanning the public chain state with one or
+// more full viewing keys.
+//
+// Wallet protocol requests include a hash of the full viewing key, used to
+// identify which set of data to query.  This also works as a pseudo-auth system
+// (assuming transport security, the client has to know the FVK to request its
+// data).  (TODO: refine this)
 service WalletProtocol {
-    // TODO: do we want to include auth/signatures in all requests here?
-    /// Get current status of chain sync
-    rpc Status() returns (StatusResult);
+    // Get current status of chain sync
+    rpc Status(crypto.FullViewingKeyHash) returns (StatusResult);
```

Comment:
Parameterizing this by the FVK hash means the service can report on sync progress of different accounts, if we decide to implement that.